### PR TITLE
Updated footer

### PR DESCRIPTION
**Pull request by ChrisHinchey**
Signed-off-by: Christopher Hinchey <chinchey@mitre.org>

### CODE REVIEW

**Comment by aaronlippold** [comment]
We may want add this to the footer of the loading model as well. Maybe to the left of version number for the github and netlify logo

<details><summary>Browser metadata</summary>





```
Path:      /
Browser:   Chrome 106.0.0.0 on Mac OS 10.15.7
Viewport:  1728 x 1000 @2x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-3617--heimdall-lite.netlify.app%2F&os=OS%20X&os_version=Catalina&browser=Chrome&browser_version=106.0&resolution=1728x1000&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-3617--heimdall-lite.netlify.app/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=633f50cc730ab200092cd34e&commentID=633f53787a7ba21ed8347c7f&resolution=resolved)

**Comment by aaronlippold** [comment]
It looks like the github icon links to the wrong repo

File: apps/frontend/src/components/global/Footer.vue [hunk 1]
```diff
@@ -3,6 +3,17 @@
...
     </span>
+    <v-spacer />
+    <a href="https://www.netlify.com">
+      <img
+        :src="require('@/assets/netlify-light.svg')"
+        style="width: 1.5rem; height: 1.5rem; vertical-align: middle"
+        class="mr-4"
+      />
+    </a>
+    <a href="https://github.com/mitre/mitre-saf" style="text-decoration: none">
```
Comment (aaronlippold) [inline apps/frontend/src/components/global/Footer.vue:14]: this should point to the heimdall2 repo right?

**Review by aaronlippold — CHANGES_REQUESTED** [review]

File: apps/frontend/src/components/global/Footer.vue (hunk 1 above)
Comment (ChrisHinchey) [reply apps/frontend/src/components/global/Footer.vue:14]: Yes, I had previously fixed that, but I had undone some changes so I must have gone back too far.  Thank you for brining that to my attention.

**Comment by ChrisHinchey** [comment]
> We may want add this to the footer of the loading model as well. Maybe to the left of version number for the github and netlify logo
> 
> Browser metadata
> [Open Deploy Preview](https://deploy-preview-3617--heimdall-lite.netlify.app/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=633f50cc730ab200092cd34e&commentID=633f53787a7ba21ed8347c7f&resolution=resolved)

File: apps/frontend/src/components/global/Footer.vue (hunk 1 above)
Comment (ChrisHinchey) [reply apps/frontend/src/components/global/Footer.vue]: Yes, I had fixed that, but I had undone something else so I must have gone back too far.  Thank you for bringing it to my attention.

Comment (aaronlippold) [reply apps/frontend/src/components/global/Footer.vue]: cool

**Comment by aaronlippold** [comment]
Looks like it wants alt text on our new images

**Comment by aaronlippold** [comment]
From the Sonarcloud scan

**Comment by ChrisHinchey** [comment]
> Looks like it wants alt text on our new images

Should be good now

**Comment by Amndeep7** [comment]
@ChrisHinchey you don't need to close a pr while you continue working on it, just mark it as a draft and then mark it as ready again when it's good to go

File: apps/frontend/src/components/global/Footer.vue [hunk 2]
```diff
@@ -3,6 +3,18 @@
...
+      <img
+        :src="require('@/assets/netlify-light.svg')"
+        style="width: 1.5rem; height: 1.5rem; vertical-align: middle"
+        class="mr-4"
+        alt="Netlify.com"
+      />
+    </a>
+    <a href="https://github.com/mitre/heimdall2" style="text-decoration: none">
+      <v-icon medium color="white">mdi-github</v-icon>
+    </a>
```
Comment (Amndeep7) [inline apps/frontend/src/components/global/Footer.vue:17]: while we're making changes here, can you add a button for releases+version number similar to the load modal.  put that button on the far right.

desired order:
left aligned: copy right notice
right aligned: github, netlify, release+version

File: apps/frontend/src/components/global/upload_tabs/HelpFooter.vue [hunk 3]
```diff
@@ -22,6 +22,14 @@
...
           <span class="d-none d-sm-inline pl-3">Github</span>
         </v-btn>
+        <v-btn href="https://www.netlify.com/" target="_blank" text small>
+          <img
+            :src="require('@/assets/netlify-light.svg')"
+            style="width: 1.15rem; height: 1.15rem"
+            alt="Netlify.com"
+          />
+          <span class="d-none d-sm-inline pl-3">Netlify</span>
+        </v-btn>
```
Comment (Amndeep7) [inline apps/frontend/src/components/global/upload_tabs/HelpFooter.vue:32]: can you change the order of the buttons to look as follows:

about, help, docs, github, netlify, release+version

**Review by Amndeep7 — CHANGES_REQUESTED** [review]
Minor order change in the buttons in both areas, then good to go imo.

**Comment by ChrisHinchey** [comment]
> @ChrisHinchey you don't need to close a pr while you continue working on it, just mark it as a draft and then mark it as ready again when it's good to go

@Amndeep7 Yeah, I had accidently clicked the close with comment button and immediately reopened it.  I wasn't aware I could convert a PR to a draft, so thank you for brining that to my attention.

**Review by Amndeep7 — APPROVED** [review]

**Pull request merged**